app/src/main/java/com/keylesspalace/tusky/BottomSheetActivity.kt

```diff
@@ -86,8 +86,11 @@ abstract class BottomSheetActivity : BaseActivity() {
                     if (statuses.isNotEmpty()) {
                         viewThread(statuses[0].id, statuses[0].url)
                         return@subscribe
-                    } else if (accounts.isNotEmpty()) {
-                        viewAccount(accounts[0].id)
+                    }
+                    accounts.firstOrNull { it.url == url }?.let { account ->
```

Comment:
(Just to make sure)

Does this url always _exactly_ match / need to?

E. g. maybe a surplus /. Or upper or lower case?

Reply:
Mmm, it *should* always match for a pasted account url. And this is not used for mentions etc., but for instances where somebody explicitly pastes `https://some.instance/@User` into a status instead of just mentioning `@User`.

In the worst case, it just opens in the browser instead of in Tusky, which is much better than the current case of "when there's no exact search result for a url, you sometimes get back random accounts and then we open one of those in Tusky instead of the url you tapped"